src/Tribe/Template_Factory.php

```diff
@@ -10,639 +10,520 @@
 	die( '-1' );
 }
 
-if ( ! class_exists( 'Tribe__Events__Template_Factory' ) ) {
-	class Tribe__Events__Template_Factory {
-
-		/**
-		 * Array of asset packages needed for this template
-		 *
-		 * @var array
-		 **/
-		protected $asset_packages = array();
-
-		/**
-		 * Length for excerpts on this template
-		 *
-		 * @var int
-		 **/
-		protected $excerpt_length = 80;
-
-		/**
-		 * Text for excerpt more on this template
-		 *
-		 * @var string
-		 **/
-		protected $excerpt_more = '&hellip;';
-
-		/**
-		 * Body class on this view
-		 *
-		 * @var string
-		 **/
-		protected $body_class = '';
-
-		/**
-		 * Static variable that holds array of vendor script handles, for adding to later deps.
-		 *
-		 * @static
-		 * @var array
-		 */
-		protected $comments_off = false;
-
-		/**
-		 * Static variable that holds array of vendor script handles, for adding to later deps.
-		 *
-		 * @static
-		 * @var array
-		 */
-		private static $vendor_scripts = array();
-
-		/**
-		 * Constant that holds the ajax hook suffix for the view
-		 *
-		 * @static
-		 * @var string
-		 */
-		const AJAX_HOOK = '';
-
-		/**
-		 * Run include packages, set up hooks
-		 *
-		 * @return void
-		 **/
-		public function __construct() {
-			$this->hooks();
-			$this->asset_packages();
-		}
+if ( class_exists( 'Tribe__Events__Template_Factory' ) ) {
+	return;
+}
 
-		/**
-		 * Set up hooks for this template
-		 *
-		 * @return void
-		 **/
-		protected function hooks() {
+class Tribe__Events__Template_Factory extends Tribe__Template_Factory {
+	/**
+	 * Length for excerpts on this template
+	 *
+	 * @var int
+	 **/
+	protected $excerpt_length = 80;
+
+	/**
+	 * Text for excerpt more on this template
+	 *
+	 * @var string
+	 **/
+	protected $excerpt_more = '&hellip;';
+
+	/**
+	 * Body class on this view
+	 *
+	 * @var string
+	 **/
+	protected $body_class = '';
+
+	/**
+	 * Static variable that holds array of vendor script handles, for adding to later deps.
+	 *
+	 * @static
+	 * @var array
+	 */
+	protected $comments_off = false;
+
+	/**
+	 * Run include packages, set up hooks
+	 *
+	 * @return void
```

Comment:
Don't really need a @return annotation for the constructor - but if we do it probably shouldn't be void